tests/features/bootstrap/FeatureContext.php

```diff
@@ -457,7 +457,11 @@ public function enableTranslationForField($field) {
    *    Fully namespaced class name.
    *
    * @throws \Behat\Mink\Exception\ExpectationException
+<<<<<<< HEAD
```

Comment:
git merge conflict generated lines, please remove and solve the conflict correctly!